mtc.go

```diff
@@ -613,7 +613,7 @@ func (w *ValidityWindow) MarshalBinary() ([]byte, error) {
 
 // Return the root recorded for this ValidityWindow's batch.
 func (w *ValidityWindow) Root() []byte {
```

Comment:
minor nit (can always address later): might be more consistent to call this `CurrHead()` and use `currHead` instead of `root` below.

Reply:
Filed #45